addons/website_slides/views/slide_channel_partner_views.xml

```diff
@@ -105,5 +105,13 @@
             </field>
         </record>
 
+        <record id="action_graduated_partner_report" model="ir.actions.server">
```

Comment:
Why do we have that server action actually ? Can't you define an act_window action instead (would replace the method in that case) ?

Reply:
Changed with an act_window